big.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-demo. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import "math/big"
+
+// BigInt wraps a golang math/big.Int.
+// All functions on BigInt have their equivalent in the documentation below.
+// See https://golang.org/pkg/math/big/#Int
+// If we do this as embedding, the functions are skipped because of the wrong return type.
+type BigInt struct {
+	i *big.Int
+}
+
+// NewBigIntFromBytes creates a BigInt from a byte slice.
+func NewBigIntFromBytes(data []byte) *BigInt {
+	return &BigInt{new(big.Int).SetBytes(data)}
+}
+
+// NewBigIntFromInt64 creates a BigInt from an int64.
+func NewBigIntFromInt64(v int64) *BigInt {
+	return &BigInt{new(big.Int).SetInt64(v)}
+}
+
+// Add wraps math/big/Int.Add
```

Comment:
```suggestion
// Add wraps math/big.Int.Add
```
same for the other methods